src/Core/tests/DeviceTests/Platforms/Windows/Package.appxmanifest

```diff
@@ -7,7 +7,7 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="2C0C8B63-C6D1-48CC-A934-5108983E28CB"
+    Name="maui-package-name-placeholder"
```

Comment:
Not hardcoding this will make it pick up the application ID from the csproj through single project